File: responses/Q3A2XR-3VW/2_sum_of_certain_multiples.py
# you can write to stdout for debugging purposes, e.g.
# print("this is a debug message")

def solution(x):
    multiples_3 = [i for i in range(0, x, 3) if i % 5 > 0]
    multiples_5 = [i for i in range(0, x, 5) if i % 3 > 0]
    return sum(multiples_3) + sum(multiples_5)

File: responses/MMVQP9-WAB/2_sum_of_certain_multiples.js
// you can write to stdout for debugging purposes, e.g.
// console.log('this is a debug message');

function solution(x) {
    var sum = 0;
    for (var a = 0; a < x; a ++) {
        if ((a % 3 == 0) && (a % 5 != 0)) {
            sum += a;
        }  else if ((a % 3 != 0) && (a % 5 == 0)) {
            sum += a;
        } 
    }
    return sum;
}